packages/ember-testing/lib/helpers/find.js

```diff
@@ -20,13 +22,16 @@ import { get } from 'ember-metal';
   ```
 
   @method find
-  @param {String} selector jQuery string selector for element lookup
+  @param {String} selector jQuery selector for element lookup
   @param {String} [context] (optional) jQuery selector that will limit the selector
                             argument to find only within the context's children
-  @return {Object} jQuery object representing the results of the query
+  @return {Object} DOM element representing the results of the query
   @public
 */
 export default function find(app, selector, context) {
+  if (jQueryDisabled) {
+    assert('If not using jQuery, please import and use methods from @ember/test-helpers');
```

Comment:
which helper exactly can be used? I think `@ember/test-helpers` doesn't provide any.

Reply:
@ro0gr - Usages of `find` should be replaced by `this.element.querySelector` or `this.element.querySelectorAll`...

Reply:
but the old good acceptance tests don't have `this.element`. Should this assertion have a reference to `setupTest` api?

Reply:
FWIW - By the time this hits a release version, the guides will all have been updated to reference the new style (work progressing in https://github.com/emberjs/guides/pull/2145).

For now, lets update the assertion to tell the user that `find` is not available when jQuery is disabled without giving a more detailed recourse (we can tweak the message when we have better docs to cross link to)...

What do y'all think?